.github/workflows/main.yml

```diff
@@ -98,10 +98,12 @@ jobs:
       branch-name: ${{ needs.build-drivers.outputs.branch-name }}
       drivers-bucket: ${{ needs.init.outputs.drivers-bucket }}
       support-packages-bucket: ${{ needs.init.outputs.support-packages-bucket }}
+      public-support-packages-bucket: ${{ needs.init.outputs.public-support-packages-bucket }}
     # If you want to test this workflow in a PR, you will need to replace
     # this condition with `${{ always() }}`, since skipping the upload of
     # bundles will cause this workflow to skip as well.
-    if: ${{ github.event_name != 'pull_request' }}
+    # if: ${{ github.event_name != 'pull_request' }}
+    if: always()
```

Comment:
just a note to revert this before merging. 

Also would it be worth adding a label to trigger this behaviour?

Reply:
Maybe we could achieve this with a condition similar to this:
```yml
if: always() && (github.event_name == 'push' || contains(github.event.pull_request.labels.*.name, 'test-support-package')
```
I think it is a little hard to read, WDYT?

Reply:
yeah I agree it's pretty complex as a condition but I think it'll make testing a lot easier and won't risk someone merging `if: always()`

Reply:
I'll add it and get it tested 🙂

Reply:
Looks like it is working properly.